### ISSUE TITLE
Rewrite entire bot with the new Message Components API

### ISSUE THREAD

**Issue by NovAti0n** [fdardenne/DiscordSINFBot#57]
Discord released the [Message Components API](https://discord.com/developers/docs/interactions/message-components) to allow developers to add buttons/dropdowns to bots. Would be nice to rewrite each command that has reactions to include buttons.

**Comment by obiwac** [comment]
There are advantages to using reactions for counting (i.e. for the poll and pin commands) rather than editing the message each time (and in the case of the poll command, it's useful to have a message which is persistent).

**Comment by NovAti0n** [comment]
Yes of course, I should've specified that in my original message. This would be perfect however for the confess command for example (commands that only need one vote: yes or no)

**Comment by obiwac** [comment]
Very true, it'd be much more ergonomic.
